src/components/Typography/Editorial.js

```diff
@@ -49,27 +49,66 @@ export const Subhead = ({ children, attributes, ...props }) => (
   </h2>
 )
 
-const lead = css({
+const leadStyle = {
   ...styles.serifRegular19,
+  lineHeight: '27px',
   margin: '0 0 10px 0',
   [mUp]: {
```

Comment:
mv `[mUp]` to `const lead`. Will be completely overwritten in `leadSmall`. If you want margin there too add it there explicitly.